common/buildcraft/factory/client/render/RenderLeds.java

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2016 AlexIIL and the BuildCraft team
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+package buildcraft.factory.client.render;
+
+import buildcraft.api.properties.BuildCraftProperties;
+import buildcraft.api.properties.BuildCraftProperty;
+import buildcraft.core.lib.utils.MathUtils;
+import buildcraft.factory.tile.ITileLed;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.block.properties.IProperty;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.model.ModelBase;
+import net.minecraft.client.model.ModelRenderer;
+import net.minecraft.client.renderer.*;
+import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.client.renderer.vertex.VertexFormat;
+import net.minecraft.entity.Entity;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.EnumSkyBlock;
+import net.minecraft.world.World;
+import net.minecraftforge.common.property.Properties;
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GL12;
+
+public class RenderLeds<T extends TileEntity & ITileLed> extends TileEntitySpecialRenderer<T> {
+    public static final RenderLeds INSTANCE = new RenderLeds();
+
+    @Override
+    public void renderTileEntityAt(T tile, double x, double y, double z, float partialTicks, int destroyStage) {
+        RenderHelper.disableStandardItemLighting();
+        GL11.glColor3f(1, 1, 1);
+        int i = 61680;
+        int j = i % 65536;
+        int k = i / 65536;
+        OpenGlHelper.setLightmapTextureCoords(OpenGlHelper.lightmapTexUnit, (float)j, (float)k);
```

Comment:
maybe add a comment where you found those magic numbers, i asume in the vanilla code so if things change you'll easely know where to find the new values